NCARDDatabase/urls.py

```diff
@@ -20,5 +20,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('ncard_app.urls')),
-
+    path('ncard_app/', include('django.contrib.auth.urls')),
```

Comment:
Remove this line referencing django.contrib.auth.urls, because we are using custom login views.